packages/sources/css/src/design-tokens/src/typography.css

```diff
@@ -21,119 +21,119 @@
  * @presenter FontSize
  */
 :root {
-  --vtmn-typo_display-1-font-size: 60px;
-  --vtmn-typo_display-2-font-size: 32px;
-  --vtmn-typo_title-1-font-size: 40px;
-  --vtmn-typo_title-2-font-size: 36px;
-  --vtmn-typo_title-3-font-size: 28px;
-  --vtmn-typo_title-4-font-size: 24px;
-  --vtmn-typo_title-5-font-size: 20px;
-  --vtmn-typo_text-1-font-size: 18px;
-  --vtmn-typo_text-2-font-size: 16px;
-  --vtmn-typo_text-3-font-size: 14px;
+  --vtmn-typo_display-1-font-size: 3.75rem;
+  --vtmn-typo_display-2-font-size: 2.625rem;
```

Comment:
is'nt it '2rem' ? 
<img width="1080" alt="Capture d’écran 2021-01-15 à 14 42 20" src="https://user-images.githubusercontent.com/46045266/104734095-00549a00-5740-11eb-8103-b7957cbc6b07.png">

Reply:
Font sizes has changed too. Please see https://www.decathlon.design/726f8c765/p/860e14-typography

Reply:
OK 👍

Reply:
Thanks @NicolasBreunin !